code/modules/mob/language.dm

```diff
@@ -499,6 +499,14 @@
 	flags = RESTRICTED | HIVEMIND | NOBABEL
 	follow = TRUE
 
+/datum/language/xenos/broadcast(mob/living/speaker, message, speaker_mask)
+	if(isalien(speaker))
+		var/mob/living/carbon/alien/humanoid/alienspeaker = speaker
+		if(alienspeaker.loudspeaker)
+			..(speaker, "<font size=3><b>[message]</b></font>")
+			return
+	..(speaker, message)
```

Comment:
```suggestion
	return ..()
```
unless you're modifying the vars, you don't need to specify the params I believe